### ISSUE TITLE
Dropdowns DO NOT open with VoiceOver iOS

### ISSUE THREAD

**Issue by cazepeda** [w3c/aria-practices#1841]
We've implemented this setup for our navigation menus which include nested menus. On VoiceOver iOS iPhone XR those nested menus DO NOT open when doing the left-to-right gesture tap. I'm sure this issue should also appear on other smaller viewport sizes and not just iPhone XR using VoiceOver.

**Comment by carmacleod** [comment]
It is possible that using `pointerover` and `pointerdown` instead of `mouseover` and `mousedown` will help with this issue.
https://developer.mozilla.org/en-US/docs/Web/API/Pointer_events

**Comment by ajanec01** [comment]
Yes, it also doesn't work on MacOS when using VO-Space bar. It's probably because there is no `click` event listener on any of the `menuitem` elements. Will test properly soon and create a pr for it.

**Comment by ajanec01** [comment]
Update: just checked and it's fixed in the editor's draft. See https://w3c.github.io/aria-practices/examples/menubar/menubar-navigation.html
